doc/maintaining/configuration.rst

```diff
@@ -655,6 +655,21 @@ Default value:  ``false``
 Controls whether the default search page (``/dataset``) should show only
 standard datasets or also custom dataset types.
 
+.. _ckan.search.default_include_private:
+
+ckan.search.default_include_private
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Example::
+
+ ckan.search.defalt_include_private = false
+
+Default value:  ``true``
+
+Controls whether the default search page (``/dataset``) should include
+private datasets visible to the current user or only public datasets
+visible to everyone.
```

Comment:
The behaviour of the API stays the same unless you're relying on `context['ignore_capacity_check'] = True` somewhere in your own code. I'd suggest replacing that with `context['user'] = site_user; data_dict['include_private'] = True`